### ISSUE TITLE
# Download the database wget http://wwwuser.gwdg.de/~compbiol/mmseqs2/tutorials/uniprot_sprot_2018_03.fasta.gz wget http://wwwuser.gwdg.de/~compbiol/mmseqs2/tutorials/uniprot_sprot_2018_03_mapping.tsv.gz gunzip uniprot_sprot_2018_03_mapping.tsv.gz  # Create a taxonomically annotated sequence database mmseqs createdb uniprot_sprot_2018_03.fasta.gz uniprot_sprot mmseqs createtaxdb uniprot_sprot tmp --tax-mapping-file uniprot_sprot_2018_03_mapping.tsv

### ISSUE THREAD

**Issue by sujee119** [soedinglab/MMseqs2#823]
Hi,

I'm trying to learn the workflow through the tutorial. unfortunately, the commands in the tutorials are wrong according to the program codes.

i followed this steps

# Download the database
wget http://wwwuser.gwdg.de/~compbiol/mmseqs2/tutorials/uniprot_sprot_2018_03.fasta.gz
wget http://wwwuser.gwdg.de/~compbiol/mmseqs2/tutorials/uniprot_sprot_2018_03_mapping.tsv.gz
gunzip uniprot_sprot_2018_03_mapping.tsv.gz

# Create a taxonomically annotated sequence database
mmseqs createdb uniprot_sprot_2018_03.fasta.gz uniprot_sprot
mmseqs createtaxdb uniprot_sprot tmp --tax-mapping-file uniprot_sprot_2018_03_mapping.tsv 

(mmseq2) cyborg@skynet:~/root_dev$ mmseqs createtaxdb uniprot_sprot tmp --tax-mapping-file uniprot_sprot_2018_03_mapping.tsv
mmseqs createtaxdb:
Annotates a sequence database with NCBI taxonomy information. The program will download the Uniprot taxMappingFile and ncbi-taxdump-folder if only two arguments are passed.

Please cite:
Steinegger, M. & Soding, J. MMseqs2 enables sensitive protein sequence searching for the analysis of massive data sets. Nature Biotechnology, doi:10.1038/nbt.3988 (2017)

© Martin Steinegger <martin.steinegger@mpibpc.mpg.de>

Usage: <i:sequenceDB> [<i:taxMappingFile> <i:ncbi-taxdump-folder>]  <tmpDir> [options]

common options	default   	description [value range]
  -v	3         	verbosity level: 0=nothing, 1: +errors, 2: +warnings, 3: +info

Unrecognized parameter --tax-mapping-file
Did you mean "-v"?

if there are any updates to the codes, please let me know.

-Thanks

**Comment by milot-mirdita** [comment]
This looks like it installed a really old version of MMseqs2. Did you install the software through conda into a new environment?

**Comment by sujee119** [comment]
> This looks like it installed a really old version of MMseqs2. Did you install the software through conda into a new environment?

Yes, i used conda to install. I also tried reinstalling and ran the codes. the error repeats.

**Comment by martin-steinegger** [comment]
Could you please share the full log? This should work! (closing it for now, please feel free to reopen if the issue persists)
